calico_node/filesystem/etc/calico/confd/config/custom_filters.cfg

```diff
@@ -0,0 +1,5 @@
+# Generated by confd
```

Comment:
OK, so according to the discussion at https://github.com/kelseyhightower/confd/issues/216, it seems you should use the `exists` keyword to handle this case.  Could you look at doing that instead?